frontend/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "frontend",
```

Comment:
Isn't this just the taker frontend perhaps?

Reply:
yes, but we are not entirely sure how to do the split yet